### ISSUE TITLE
Rogue `console.log` on npm

### ISSUE THREAD

**Issue by gnestor** [withspectrum/draft-js-prism-plugin#5]
I saw https://github.com/withspectrum/draft-js-prism-plugin/commit/e7a8dbcd4ec3271fc25c5c54bfe36f36c6247fec which removes the rogue `console.log` from the source, but if you do the following you'll see it in there:

```
$ npm pack draft-js-prism-plugin
draft-js-prism-plugin-0.1.1.tgz
$ tar -xvzf draft-js-prism-plugin-0.1.1.tgz
$ cat package/dist/index.js 
'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});

var _typeof = typeof Symbol === "function" && typeof Symbol.iterator === "symbol" ? function (obj) { return typeof obj; } : function (obj) { return obj && typeof Symbol === "function" && obj.constructor === Symbol && obj !== Symbol.prototype ? "symbol" : typeof obj; };

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

var _draftJsPrism = require('draft-js-prism');

var _draftJsPrism2 = _interopRequireDefault(_draftJsPrism);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

var prismPlugin = function prismPlugin(options) {
  if (!options || !options.prism) throw new Error('You must provide an instance of the prism.js library to draft-js-prism-plugin. See the documentation at https://github.com/withspectrum/draft-js-prism-plugin for more information!');
  var prism = options.prism;

  return {
    decorators: [new _draftJsPrism2.default({
      prism: prism,
      getSyntax: function getSyntax(block) {
        var language = block.getData().get('language');
        console.log(language);
        if (_typeof(prism.languages[language]) === 'object') {
          return language;
        }
        return null;
      },
      render: function render(_ref) {
        var type = _ref.type,
            children = _ref.children;

        return _react2.default.createElement(
          'span',
          { className: 'prism-token token ' + type },
          children
        );
      }
    })]
  };
};
```

Can you rebuild and publish a new version to npm please? 🙂

**Comment by mxstbr** [comment]
Arg, yep!

**Comment by gnestor** [comment]
Can you just publish a new version from master?

**Comment by mxstbr** [comment]
Ah, I totally forgot. Will do right now, one sec! Sorry for the delay, thanks for the reminder 🙏

**Comment by mxstbr** [comment]
Published in v0.1.2, sorry for the delay!

**Comment by gnestor** [comment]
Thanks @mxstbr!
